ui/package-lock.json

```diff
@@ -54,16 +54,16 @@
       "version": "1.0.0",
```

Comment:
Should be no changes to this file. Make sure patch-db and other deps are up to date.